src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,73 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String report = this.generateReport(fromFileName);
+        this.writeToFile(report, toFileName);
+        System.out.println(report);
```

Comment:
```suggestion
        List<String> dataFromFile = readFromFile(fromFileName);
        String report = createReport(dataFromFile);
        writeToFile(toFileName, report);
```
program should look like this